### PR TITLE
Add Secta adapter

### PR DESCRIPTION
**Pull request by jac0x**
Secta Finance

DEX uniswap-style v2 & v3

### CODE REVIEW

File: adapters/secta/src/sdk/positionDetails.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    
+
+    if (currentTick < tickLow) {
+        amount0 = BigInt(Math.floor(Number(liquidity) * ((sqrtRatioB - sqrtRatioA) / (sqrtRatioA * sqrtRatioB))));
+    } else if (currentTick >= tickHigh) {
+        amount1 = BigInt(Math.floor(Number(liquidity) * (sqrtRatioB - sqrtRatioA)));
+    } else if (currentTick >= tickLow && currentTick < tickHigh) {
+        amount0 = BigInt(Math.floor(Number(liquidity) * ((sqrtRatioB - sqrtPrice) / (sqrtPrice * sqrtRatioB))));
+        amount1 = BigInt(Math.floor(Number(liquidity) * (sqrtPrice - sqrtRatioA)));
+    }
```
Comment (0xroll) [inline adapters/secta/src/sdk/positionDetails.ts:42]: hi, we are looking to remove inactive liquidity.

can you edit this part to not calculate any position that are out of range?

**Comment by 0xroll** [comment]
Hi @lijac thanks for the PR.

for linea points calculation, we are only considering LP positions that are active (in range).

can you make changes to the codes ? you can also refer to sushi's PR : https://github.com/delta-hq/l2-lxp-liquidity-reward/pull/37/files

File: adapters/secta/src/sdk/positionDetails.ts (hunk 1 above)
Comment (jac0x) [reply adapters/secta/src/sdk/positionDetails.ts]: I've removed first and second if statements.

**Comment by jac0x** [comment]
@0xroll I'm late, thanks for review.

I've updated code. How can I see if this is working in action? Is there a dashboard on your side?

From performance perspective, I see sushi's code is much simpler than ours. Is it a big deal?

**Comment by 0xroll** [comment]
> @0xroll I'm late, thanks for review.
> 
> I've updated code. How can I see if this is working in action? Is there a dashboard on your side?
> 
> From performance perspective, I see sushi's code is much simpler than ours. Is it a big deal?

hey please check https://github.com/delta-hq/l2-lxp-liquidity-reward/issues/95

**Pull request merged**